File: packages/core/test/fixtures/regex/csharp/launchdarkly/positive/basic.cs
using LaunchDarkly.Sdk.Server;

public class Checkout
{
    private readonly LdClient client;

    public Checkout(LdClient client)
    {
        this.client = client;
    }

    public string Run(Context context)
    {
        bool enabled = client.BoolVariation("CHECKOUT_V2", context, false);
        if (enabled)
        {
            return "v2";
        }
        return "v1";
    }
}

File: packages/core/test/fixtures/regex/ruby/launchdarkly/positive/basic.rb
require 'launchdarkly-server-sdk'

class Checkout
  def initialize(client)
    @client = client
  end

  def run(context)
    enabled = @client.variation("CHECKOUT_V2", context, false)
    if enabled
      return "v2"
    end
    "v1"
  end
end
